apps/frontend/src/components/navbar.tsx

```diff
@@ -0,0 +1,43 @@
+import Image from 'next/image';
+
+import styles from '@/app/globals.module.css';
+
+function Navbar() {
+  return (
+    <nav className={styles.navbar}>
```

Comment:
Try to keep the code consistent by using the same code structure for the same things.
For example don't use
styles.navbar and styles['link'] simultaneously. 
styles['logo-container'] is a different story, but we could consider a different name for these, e.g. using camel case.